apps/s_server.c

```diff
@@ -1728,6 +1759,23 @@ int s_server_main(int argc, char *argv[])
     if (!load_excert(&exc))
         goto end;
 
+#ifndef OPENSSL_NO_RPK
+    if (peer_rpk_file != NULL) {
+        peer_rpk = load_pubkey(peer_rpk_file, s_key_format, 0, pass, engine, "server RPK file");
+        if (peer_rpk == NULL) {
+            peer_rpk_cert = load_cert(peer_rpk_file, s_cert_format, "server RPK file");
```

Comment:
I think these strings should also be "client RPK file", since the peer of `s_server` is the client.